api/pyproject.toml

```diff
@@ -24,7 +24,6 @@ pre-commit = "^2.19.0"
 flake8 = "^4.0.1"
 black = "^22.3.0"
 isort = "^5.10.1"
-interrogate = "^1.5.0"
 bandit = "^1.7.4"
```

Comment:
What was the reason for removing interrogate? Is it considered insecure by pip-audit? Interesting.

Reply:
interrogate in itself is not insecure, but it uses "py", which is.